{"uid":"67318868f08274d8","name":"Test Delete Admin Agent","fullName":"test_cases.test_agent.TestAgentManagement#test_delete_admin_agent","historyId":"bf5102b0fa26be747eb8ee000c1d24e7","time":{"start":1727152501864,"stop":1727152501864,"duration":0},"status":"passed","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[{"name":"autonomous_agent_api","time":{"start":1727152014606,"stop":1727152014606,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false},{"name":"admin_login_response","time":{"start":1727152015194,"stop":1727152015317,"duration":123},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false},{"name":"create_template_fixture","time":{"start":1727152015317,"stop":1727152015444,"duration":127},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false},{"name":"admin_login_cookie","time":{"start":1727152015317,"stop":1727152015317,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false},{"name":"edit_template_fixture","time":{"start":1727152015444,"stop":1727152015579,"duration":135},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false},{"name":"create_admin_agent_fixture","time":{"start":1727152015579,"stop":1727152015717,"duration":138},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false},{"name":"edit_admin_agent_fixture","time":{"start":1727152015849,"stop":1727152015996,"duration":147},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false},{"name":"load_funds_to_agent_address","time":{"start":1727152015997,"stop":1727152137414,"duration":121417},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false},{"name":"run_admin_agent_fixture","time":{"start":1727152137414,"stop":1727152501335,"duration":363921},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false},{"name":"delete_admin_agent_fixture","time":{"start":1727152501336,"stop":1727152501863,"duration":527},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false}],"testStage":{"status":"passed","steps":[],"attachments":[{"uid":"4d044ce3893bf88f","name":"log","source":"4d044ce3893bf88f.txt","type":"text/plain","size":305},{"uid":"1938cc9e2d8efd9a","name":"stdout","source":"1938cc9e2d8efd9a.txt","type":"text/plain","size":451},{"uid":"a035d24f2b96af70","name":"stderr","source":"a035d24f2b96af70.txt","type":"text/plain","size":1137}],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":3,"hasContent":true},"afterStages":[{"name":"autonomous_agent_api::0","time":{"start":1727152503285,"stop":1727152503285,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false}],"labels":[{"name":"parentSuite","value":"Agent Management"},{"name":"feature","value":"Delete Admin Agent"},{"name":"suite","value":"test_agent"},{"name":"subSuite","value":"TestAgentManagement"},{"name":"host","value":"fv-az738-566"},{"name":"thread","value":"1924-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"test_cases.test_agent"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[],"categories":[],"history":{"statistic":{"failed":10,"broken":0,"skipped":0,"passed":25,"unknown":0,"total":35},"items":[{"uid":"8b78a99ed7a7628e","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/34//#testresult/8b78a99ed7a7628e","status":"passed","time":{"start":1727148732498,"stop":1727148732498,"duration":0}},{"uid":"d1b1ff7668a192c4","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/34//#testresult/d1b1ff7668a192c4","status":"failed","statusDetails":"AssertionError: Expected DELETE /agents/cbccb0ca-8534-47ca-95e0-0020248e9820 to succeed but got statusCode: 502 : body: <html>\r\n<head><title>502 Bad Gateway</title></head>\r\n<body>\r\n<center><h1>502 Bad Gateway</h1></center>\r\n<hr><center>nginx/1.23.2</center>\r\n</body>\r\n</html>","time":{"start":1727087529711,"stop":1727087529711,"duration":0}},{"uid":"9a4807b546197b6c","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/33//#testresult/9a4807b546197b6c","status":"failed","statusDetails":"AssertionError: Expected POST /templates to succeed but got statusCode: 403 : body: \"Forbidden Request\"","time":{"start":1726811571608,"stop":1726811571608,"duration":0}},{"uid":"7ad5abed68ee7076","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/32//#testresult/7ad5abed68ee7076","status":"failed","statusDetails":"AssertionError: Expected POST /templates to succeed but got statusCode: 403 : body: \"Forbidden Request\"","time":{"start":1726738987747,"stop":1726738987747,"duration":0}},{"uid":"1668de7329c98266","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/31//#testresult/1668de7329c98266","status":"failed","statusDetails":"AssertionError: Expected POST /templates to succeed but got statusCode: 403 : body: \"Forbidden Request\"","time":{"start":1726730682599,"stop":1726730682599,"duration":0}},{"uid":"ec882a94aecb03fe","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/30//#testresult/ec882a94aecb03fe","status":"passed","time":{"start":1726730663450,"stop":1726730663450,"duration":0}},{"uid":"426b1badecc9458b","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/29//#testresult/426b1badecc9458b","status":"passed","time":{"start":1726655637826,"stop":1726655637827,"duration":1}},{"uid":"3ed0b4844decbb54","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/28//#testresult/3ed0b4844decbb54","status":"passed","time":{"start":1726572393689,"stop":1726572393689,"duration":0}},{"uid":"544c4eb34eafa9d6","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/27//#testresult/544c4eb34eafa9d6","status":"passed","time":{"start":1726571193182,"stop":1726571193182,"duration":0}},{"uid":"4d44cd4ed3144eea","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/26//#testresult/4d44cd4ed3144eea","status":"passed","time":{"start":1726555652878,"stop":1726555652878,"duration":0}},{"uid":"b2c41dfbaf75dd00","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/25//#testresult/b2c41dfbaf75dd00","status":"passed","time":{"start":1726554683557,"stop":1726554683557,"duration":0}},{"uid":"5527423b23ff0a5","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/24//#testresult/5527423b23ff0a5","status":"passed","time":{"start":1726552149495,"stop":1726552149495,"duration":0}},{"uid":"f09407a36b3eee27","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/23//#testresult/f09407a36b3eee27","status":"passed","time":{"start":1726475579615,"stop":1726475579616,"duration":1}},{"uid":"e7fc9329dd2d9ed5","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/22//#testresult/e7fc9329dd2d9ed5","status":"passed","time":{"start":1726461465537,"stop":1726461465537,"duration":0}},{"uid":"627e4c8c897ce279","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/19//#testresult/627e4c8c897ce279","status":"failed","statusDetails":"AssertionError: Expected PUT /agents/0016a154-7e9f-4729-a27d-fce73da9fdbc to succeed but got statusCode: 500 : body: Traceback (most recent call last):\n  File \"/usr/local/lib/python3.12/site-packages/starlette/middleware/errors.py\", line 162, in __call__\n    await self.app(scope, receive, _send)\n  File \"/usr/local/lib/python3.12/site-packages/starlette/middleware/cors.py\", line 83, in __call__\n    await self.app(scope, receive, send)\n  File \"/usr/local/lib/python3.12/site-packages/starlette/middleware/exceptions.py\", line 79, in __call__\n    raise exc\n  File \"/usr/local/lib/python3.12/site-packages/starlette/middleware/exceptions.py\", line 68, in __call__\n    await self.app(scope, receive, sender)\n  File \"/usr/local/lib/python3.12/site-packages/fastapi/middleware/asyncexitstack.py\", line 20, in __call__\n    raise e\n  File \"/usr/local/lib/python3.12/site-packages/fastapi/middleware/asyncexitstack.py\", line 17, in __call__\n    await self.app(scope, receive, send)\n  File \"/usr/local/lib/python3.12/site-packages/starlette/routing.py\", line 718, in __call__\n    await route.handle(scope, receive, send)\n  File \"/usr/local/lib/python3.12/site-packages/starlette/routing.py\", line 276, in handle\n    await self.app(scope, receive, send)\n  File \"/usr/local/lib/python3.12/site-packages/starlette/routing.py\", line 66, in app\n    response = await func(request)\n               ^^^^^^^^^^^^^^^^^^^\n  File \"/usr/local/lib/python3.12/site-packages/fastapi/routing.py\", line 241, in app\n    raw_response = await run_endpoint_function(\n                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/usr/local/lib/python3.12/site-packages/fastapi/routing.py\", line 167, in run_endpoint_function\n    return await dependant.call(**values)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/api/backend/backend/app/controllers/agent_router.py\", line 48, in update_agent\n    updated_agent = await self.agent_service.update_agent(agent_id, agent_data, userAddress=user.address)\n                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/api/backend/backend/app/services/agent_service.py\", line 100, in update_agent\n    if existing_agent.instance < agent_data.instance:\n       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\nTypeError: '<' not supported between instances of 'int' and 'NoneType'","time":{"start":1726056110333,"stop":1726056110333,"duration":0}},{"uid":"28cf64adad376add","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/18//#testresult/28cf64adad376add","status":"failed","statusDetails":"AssertionError: Expected PUT /agents/52ee8252-01f5-427b-9777-842bdbc37d18 to succeed but got statusCode: 500 : body: Traceback (most recent call last):\n  File \"/usr/local/lib/python3.12/site-packages/starlette/middleware/errors.py\", line 162, in __call__\n    await self.app(scope, receive, _send)\n  File \"/usr/local/lib/python3.12/site-packages/starlette/middleware/cors.py\", line 83, in __call__\n    await self.app(scope, receive, send)\n  File \"/usr/local/lib/python3.12/site-packages/starlette/middleware/exceptions.py\", line 79, in __call__\n    raise exc\n  File \"/usr/local/lib/python3.12/site-packages/starlette/middleware/exceptions.py\", line 68, in __call__\n    await self.app(scope, receive, sender)\n  File \"/usr/local/lib/python3.12/site-packages/fastapi/middleware/asyncexitstack.py\", line 20, in __call__\n    raise e\n  File \"/usr/local/lib/python3.12/site-packages/fastapi/middleware/asyncexitstack.py\", line 17, in __call__\n    await self.app(scope, receive, send)\n  File \"/usr/local/lib/python3.12/site-packages/starlette/routing.py\", line 718, in __call__\n    await route.handle(scope, receive, send)\n  File \"/usr/local/lib/python3.12/site-packages/starlette/routing.py\", line 276, in handle\n    await self.app(scope, receive, send)\n  File \"/usr/local/lib/python3.12/site-packages/starlette/routing.py\", line 66, in app\n    response = await func(request)\n               ^^^^^^^^^^^^^^^^^^^\n  File \"/usr/local/lib/python3.12/site-packages/fastapi/routing.py\", line 241, in app\n    raw_response = await run_endpoint_function(\n                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/usr/local/lib/python3.12/site-packages/fastapi/routing.py\", line 167, in run_endpoint_function\n    return await dependant.call(**values)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/api/backend/backend/app/controllers/agent_router.py\", line 48, in update_agent\n    updated_agent = await self.agent_service.update_agent(agent_id, agent_data, userAddress=user.address)\n                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/api/backend/backend/app/services/agent_service.py\", line 100, in update_agent\n    if existing_agent.instance < agent_data.instance:\n       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\nTypeError: '<' not supported between instances of 'int' and 'NoneType'","time":{"start":1726051674581,"stop":1726051674581,"duration":0}},{"uid":"1f880bfce7aeabef","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/17//#testresult/1f880bfce7aeabef","status":"failed","statusDetails":"AssertionError: Expected PUT /agents/bc73d298-36ba-42e2-b8ba-889292fd3a0f to succeed but got statusCode: 500 : body: Traceback (most recent call last):\n  File \"/usr/local/lib/python3.12/site-packages/starlette/middleware/errors.py\", line 162, in __call__\n    await self.app(scope, receive, _send)\n  File \"/usr/local/lib/python3.12/site-packages/starlette/middleware/cors.py\", line 83, in __call__\n    await self.app(scope, receive, send)\n  File \"/usr/local/lib/python3.12/site-packages/starlette/middleware/exceptions.py\", line 79, in __call__\n    raise exc\n  File \"/usr/local/lib/python3.12/site-packages/starlette/middleware/exceptions.py\", line 68, in __call__\n    await self.app(scope, receive, sender)\n  File \"/usr/local/lib/python3.12/site-packages/fastapi/middleware/asyncexitstack.py\", line 20, in __call__\n    raise e\n  File \"/usr/local/lib/python3.12/site-packages/fastapi/middleware/asyncexitstack.py\", line 17, in __call__\n    await self.app(scope, receive, send)\n  File \"/usr/local/lib/python3.12/site-packages/starlette/routing.py\", line 718, in __call__\n    await route.handle(scope, receive, send)\n  File \"/usr/local/lib/python3.12/site-packages/starlette/routing.py\", line 276, in handle\n    await self.app(scope, receive, send)\n  File \"/usr/local/lib/python3.12/site-packages/starlette/routing.py\", line 66, in app\n    response = await func(request)\n               ^^^^^^^^^^^^^^^^^^^\n  File \"/usr/local/lib/python3.12/site-packages/fastapi/routing.py\", line 241, in app\n    raw_response = await run_endpoint_function(\n                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/usr/local/lib/python3.12/site-packages/fastapi/routing.py\", line 167, in run_endpoint_function\n    return await dependant.call(**values)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/api/backend/backend/app/controllers/agent_router.py\", line 48, in update_agent\n    updated_agent = await self.agent_service.update_agent(agent_id, agent_data, userAddress=user.address)\n                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/api/backend/backend/app/services/agent_service.py\", line 96, in update_agent\n    existing_agent = await self.agent_repository.modify_agent(agent_id, agent_data)\n                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/api/backend/backend/app/repositories/agent_repository.py\", line 80, in modify_agent\n    updated_agent = await self.db.prisma.agent.update(where={\"id\": agent_id}, data=updated_data)\n                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/usr/local/lib/python3.12/site-packages/prisma/actions.py\", line 666, in update\n    resp = await self._client._execute(\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/usr/local/lib/python3.12/site-packages/prisma/_base_client.py\", line 533, in _execute\n    return await self._engine.query(builder.build(), tx_id=self._tx_id)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/usr/local/lib/python3.12/site-packages/prisma/engine/_query.py\", line 404, in query\n    return await self.request(\n           ^^^^^^^^^^^^^^^^^^^\n  File \"/usr/local/lib/python3.12/site-packages/prisma/engine/_http.py\", line 233, in request\n    return self._process_response_data(data=data, response=response)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/usr/local/lib/python3.12/site-packages/prisma/engine/_http.py\", line 87, in _process_response_data\n    return utils.handle_response_errors(response, errors_data)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/usr/local/lib/python3.12/site-packages/prisma/engine/utils.py\", line 163, in handle_response_errors\n    raise prisma_errors.MissingRequiredValueError(error)\nprisma.errors.MissingRequiredValueError: Unable to match input value to any allowed input type for the field. Parse errors: [Unable to match input value to any allowed input type for the field. Parse errors: [`data.instance`: A value is required but not set, `data.instance`: A value is required but not set], Unable to match input value to any allowed input type for the field. Parse errors: [`data.instance`: A value is required but not set, `data.instance`: A value is required but not set]]","time":{"start":1726047884375,"stop":1726047884375,"duration":0}},{"uid":"a3fadd300fb153c1","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/16//#testresult/a3fadd300fb153c1","status":"failed","statusDetails":"AssertionError: Expected PUT /agents/e88495d5-40a1-4f1d-8d73-c8988548d179 to succeed but got statusCode: 500 : body: Traceback (most recent call last):\n  File \"/usr/local/lib/python3.12/site-packages/starlette/middleware/errors.py\", line 162, in __call__\n    await self.app(scope, receive, _send)\n  File \"/usr/local/lib/python3.12/site-packages/starlette/middleware/cors.py\", line 83, in __call__\n    await self.app(scope, receive, send)\n  File \"/usr/local/lib/python3.12/site-packages/starlette/middleware/exceptions.py\", line 79, in __call__\n    raise exc\n  File \"/usr/local/lib/python3.12/site-packages/starlette/middleware/exceptions.py\", line 68, in __call__\n    await self.app(scope, receive, sender)\n  File \"/usr/local/lib/python3.12/site-packages/fastapi/middleware/asyncexitstack.py\", line 20, in __call__\n    raise e\n  File \"/usr/local/lib/python3.12/site-packages/fastapi/middleware/asyncexitstack.py\", line 17, in __call__\n    await self.app(scope, receive, send)\n  File \"/usr/local/lib/python3.12/site-packages/starlette/routing.py\", line 718, in __call__\n    await route.handle(scope, receive, send)\n  File \"/usr/local/lib/python3.12/site-packages/starlette/routing.py\", line 276, in handle\n    await self.app(scope, receive, send)\n  File \"/usr/local/lib/python3.12/site-packages/starlette/routing.py\", line 66, in app\n    response = await func(request)\n               ^^^^^^^^^^^^^^^^^^^\n  File \"/usr/local/lib/python3.12/site-packages/fastapi/routing.py\", line 241, in app\n    raw_response = await run_endpoint_function(\n                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/usr/local/lib/python3.12/site-packages/fastapi/routing.py\", line 167, in run_endpoint_function\n    return await dependant.call(**values)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/api/backend/backend/app/controllers/agent_router.py\", line 48, in update_agent\n    updated_agent = await self.agent_service.update_agent(agent_id, agent_data, userAddress=user.address)\n                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/api/backend/backend/app/services/agent_service.py\", line 96, in update_agent\n    existing_agent = await self.agent_repository.modify_agent(agent_id, agent_data)\n                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/api/backend/backend/app/repositories/agent_repository.py\", line 80, in modify_agent\n    updated_agent = await self.db.prisma.agent.update(where={\"id\": agent_id}, data=updated_data)\n                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/usr/local/lib/python3.12/site-packages/prisma/actions.py\", line 666, in update\n    resp = await self._client._execute(\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/usr/local/lib/python3.12/site-packages/prisma/_base_client.py\", line 533, in _execute\n    return await self._engine.query(builder.build(), tx_id=self._tx_id)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/usr/local/lib/python3.12/site-packages/prisma/engine/_query.py\", line 404, in query\n    return await self.request(\n           ^^^^^^^^^^^^^^^^^^^\n  File \"/usr/local/lib/python3.12/site-packages/prisma/engine/_http.py\", line 233, in request\n    return self._process_response_data(data=data, response=response)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/usr/local/lib/python3.12/site-packages/prisma/engine/_http.py\", line 87, in _process_response_data\n    return utils.handle_response_errors(response, errors_data)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/usr/local/lib/python3.12/site-packages/prisma/engine/utils.py\", line 163, in handle_response_errors\n    raise prisma_errors.MissingRequiredValueError(error)\nprisma.errors.MissingRequiredValueError: Unable to match input value to any allowed input type for the field. Parse errors: [Unable to match input value to any allowed input type for the field. Parse errors: [`data.instance`: A value is required but not set, `data.instance`: A value is required but not set], Unable to match input value to any allowed input type for the field. Parse errors: [`data.instance`: A value is required but not set, `data.instance`: A value is required but not set]]","time":{"start":1726037915625,"stop":1726037915625,"duration":0}},{"uid":"160f6b75b17a2ecd","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/15//#testresult/160f6b75b17a2ecd","status":"failed","statusDetails":"AssertionError: Expected PUT /agents/b9e0c627-e84e-49c9-b257-7595ede11aa7 to succeed but got statusCode: 500 : body: Traceback (most recent call last):\n  File \"/usr/local/lib/python3.12/site-packages/starlette/middleware/errors.py\", line 162, in __call__\n    await self.app(scope, receive, _send)\n  File \"/usr/local/lib/python3.12/site-packages/starlette/middleware/cors.py\", line 83, in __call__\n    await self.app(scope, receive, send)\n  File \"/usr/local/lib/python3.12/site-packages/starlette/middleware/exceptions.py\", line 79, in __call__\n    raise exc\n  File \"/usr/local/lib/python3.12/site-packages/starlette/middleware/exceptions.py\", line 68, in __call__\n    await self.app(scope, receive, sender)\n  File \"/usr/local/lib/python3.12/site-packages/fastapi/middleware/asyncexitstack.py\", line 20, in __call__\n    raise e\n  File \"/usr/local/lib/python3.12/site-packages/fastapi/middleware/asyncexitstack.py\", line 17, in __call__\n    await self.app(scope, receive, send)\n  File \"/usr/local/lib/python3.12/site-packages/starlette/routing.py\", line 718, in __call__\n    await route.handle(scope, receive, send)\n  File \"/usr/local/lib/python3.12/site-packages/starlette/routing.py\", line 276, in handle\n    await self.app(scope, receive, send)\n  File \"/usr/local/lib/python3.12/site-packages/starlette/routing.py\", line 66, in app\n    response = await func(request)\n               ^^^^^^^^^^^^^^^^^^^\n  File \"/usr/local/lib/python3.12/site-packages/fastapi/routing.py\", line 241, in app\n    raw_response = await run_endpoint_function(\n                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/usr/local/lib/python3.12/site-packages/fastapi/routing.py\", line 167, in run_endpoint_function\n    return await dependant.call(**values)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/api/backend/backend/app/controllers/agent_router.py\", line 48, in update_agent\n    updated_agent = await self.agent_service.update_agent(agent_id, agent_data, userAddress=user.address)\n                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/api/backend/backend/app/services/agent_service.py\", line 96, in update_agent\n    existing_agent = await self.agent_repository.modify_agent(agent_id, agent_data)\n                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/api/backend/backend/app/repositories/agent_repository.py\", line 80, in modify_agent\n    updated_agent = await self.db.prisma.agent.update(where={\"id\": agent_id}, data=updated_data)\n                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/usr/local/lib/python3.12/site-packages/prisma/actions.py\", line 666, in update\n    resp = await self._client._execute(\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/usr/local/lib/python3.12/site-packages/prisma/_base_client.py\", line 533, in _execute\n    return await self._engine.query(builder.build(), tx_id=self._tx_id)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/usr/local/lib/python3.12/site-packages/prisma/engine/_query.py\", line 404, in query\n    return await self.request(\n           ^^^^^^^^^^^^^^^^^^^\n  File \"/usr/local/lib/python3.12/site-packages/prisma/engine/_http.py\", line 233, in request\n    return self._process_response_data(data=data, response=response)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/usr/local/lib/python3.12/site-packages/prisma/engine/_http.py\", line 87, in _process_response_data\n    return utils.handle_response_errors(response, errors_data)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/usr/local/lib/python3.12/site-packages/prisma/engine/utils.py\", line 163, in handle_response_errors\n    raise prisma_errors.MissingRequiredValueError(error)\nprisma.errors.MissingRequiredValueError: Unable to match input value to any allowed input type for the field. Parse errors: [Unable to match input value to any allowed input type for the field. Parse errors: [`data.instance`: A value is required but not set, `data.instance`: A value is required but not set], Unable to match input value to any allowed input type for the field. Parse errors: [`data.instance`: A value is required but not set, `data.instance`: A value is required but not set]]","time":{"start":1726031419944,"stop":1726031419944,"duration":0}},{"uid":"77598ddf2a135c1b","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/14//#testresult/77598ddf2a135c1b","status":"failed","statusDetails":"AssertionError: Expected PUT /agents/32a5ddcb-4aec-4cae-898e-9ca7701cf69b to succeed but got statusCode: 500 : body: Traceback (most recent call last):\n  File \"/usr/local/lib/python3.12/site-packages/starlette/middleware/errors.py\", line 162, in __call__\n    await self.app(scope, receive, _send)\n  File \"/usr/local/lib/python3.12/site-packages/starlette/middleware/cors.py\", line 83, in __call__\n    await self.app(scope, receive, send)\n  File \"/usr/local/lib/python3.12/site-packages/starlette/middleware/exceptions.py\", line 79, in __call__\n    raise exc\n  File \"/usr/local/lib/python3.12/site-packages/starlette/middleware/exceptions.py\", line 68, in __call__\n    await self.app(scope, receive, sender)\n  File \"/usr/local/lib/python3.12/site-packages/fastapi/middleware/asyncexitstack.py\", line 20, in __call__\n    raise e\n  File \"/usr/local/lib/python3.12/site-packages/fastapi/middleware/asyncexitstack.py\", line 17, in __call__\n    await self.app(scope, receive, send)\n  File \"/usr/local/lib/python3.12/site-packages/starlette/routing.py\", line 718, in __call__\n    await route.handle(scope, receive, send)\n  File \"/usr/local/lib/python3.12/site-packages/starlette/routing.py\", line 276, in handle\n    await self.app(scope, receive, send)\n  File \"/usr/local/lib/python3.12/site-packages/starlette/routing.py\", line 66, in app\n    response = await func(request)\n               ^^^^^^^^^^^^^^^^^^^\n  File \"/usr/local/lib/python3.12/site-packages/fastapi/routing.py\", line 241, in app\n    raw_response = await run_endpoint_function(\n                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/usr/local/lib/python3.12/site-packages/fastapi/routing.py\", line 167, in run_endpoint_function\n    return await dependant.call(**values)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/api/backend/backend/app/controllers/agent_router.py\", line 48, in update_agent\n    updated_agent = await self.agent_service.update_agent(agent_id, agent_data, userAddress=user.address)\n                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/api/backend/backend/app/services/agent_service.py\", line 96, in update_agent\n    existing_agent = await self.agent_repository.modify_agent(agent_id, agent_data)\n                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/api/backend/backend/app/repositories/agent_repository.py\", line 80, in modify_agent\n    updated_agent = await self.db.prisma.agent.update(where={\"id\": agent_id}, data=updated_data)\n                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/usr/local/lib/python3.12/site-packages/prisma/actions.py\", line 666, in update\n    resp = await self._client._execute(\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/usr/local/lib/python3.12/site-packages/prisma/_base_client.py\", line 533, in _execute\n    return await self._engine.query(builder.build(), tx_id=self._tx_id)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/usr/local/lib/python3.12/site-packages/prisma/engine/_query.py\", line 404, in query\n    return await self.request(\n           ^^^^^^^^^^^^^^^^^^^\n  File \"/usr/local/lib/python3.12/site-packages/prisma/engine/_http.py\", line 233, in request\n    return self._process_response_data(data=data, response=response)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/usr/local/lib/python3.12/site-packages/prisma/engine/_http.py\", line 87, in _process_response_data\n    return utils.handle_response_errors(response, errors_data)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/usr/local/lib/python3.12/site-packages/prisma/engine/utils.py\", line 163, in handle_response_errors\n    raise prisma_errors.MissingRequiredValueError(error)\nprisma.errors.MissingRequiredValueError: Unable to match input value to any allowed input type for the field. Parse errors: [Unable to match input value to any allowed input type for the field. Parse errors: [`data.instance`: A value is required but not set, `data.instance`: A value is required but not set], Unable to match input value to any allowed input type for the field. Parse errors: [`data.instance`: A value is required but not set, `data.instance`: A value is required but not set]]","time":{"start":1725963147569,"stop":1725963147569,"duration":0}}]},"tags":[]},"source":"67318868f08274d8.json","parameterValues":[]}